src/docs/devdoc-index.dox

```diff
@@ -1,5 +1,10 @@
 /*! @page devdoc-index Developer Documentation
 
+@subpage devdoc-glossary
+
+WiredTiger assigns specific meanings to some words, the following page decodes
```

Comment:
Technically, there is no following page as the link is before the text. Maybe something like _"WiredTiger assigns meanings to specific meanings to certain words. Here we decode them."_  Or similar?